marketdata/stream/entities.go

```diff
@@ -12,6 +12,17 @@ type Trade struct {
 	Timestamp  time.Time
 	Conditions []string
 	Tape       string
+
+	internal tradeInternal
+}
+
+type tradeInternal struct {
+	ReceivedAt time.Time
+}
+
+// Internal contains internal fields.
```

Comment:
Please expand on this comment: "Do not expect any of the fields to be set and no future compatibility is promised for this struct. You should not use this at all" or something similar. The current comment is not scary enough.